apps/test/unit/templates/rubrics/AiAssessmentBoxTest.jsx

```diff
@@ -15,11 +15,7 @@ import {RubricUnderstandingLevels} from '@cdo/generated-scripts/sharedConstants'
 import i18n from '@cdo/locale';
 
 async function wait() {
-  for (let _ = 0; _ < 10; _++) {
-    await act(async () => {
-      await Promise.resolve();
-    });
-  }
+  await act(async () => Promise.resolve());
```

Comment:
You may have to wait more than once, in this case, if you're trying to get through a few `await` lines. The `act` will wait until Promises resolve... but basically, they might resolve into another Promise. Firefox for some reason generally needs one extra `act` over Chrome (although admittedly I think I'm the only one that runs JS tests on Firefox).

I wonder how much Jest changes this if it isn't targeting a specific browser implementation... haven't tried it thoroughly since then. 🤷

Reply:
I agree that this might not work in all cases, but since this is limited to this one file, I thought it might be ok? If we ran into a problem when adding another test, the test writer could always work around it by calling `wait()` a second time, perhaps commenting on what each one is waiting for. this seems better to me than blindly waiting 10 times. this also reads more simply: "wait one network request, then update the DOM". I suppose i could add that in a comment!

I don't think we'll have to worry about other browsers, now that we are running the tests in Jest.

Reply:
Agreed!